portfolio/src/main/webapp/index.html

```diff
@@ -11,157 +11,187 @@
   </head>
   <body>
     <!--Navbar-->
-    <div class="navbar">
+    <div id="navbar" class="navbar navbar-expand-lg">
```

Comment:
Looks like there are a lot of line diffs in this file - but I guess it's mostly due to a change in indentation/formatting of the HTML?

Reply:
Yes unfortunately. I just added bootstrap div rows and columns to create the side bar.

Reply:
Got it, no problem!